src/WordSearch/WordSearchPoint.ts

```diff
@@ -0,0 +1,8 @@
+export class WordSearchPoint {

```

Comment:
"WordSearchPoint" seems fairly verbose given that this is in the Word Search directory.  I'd be tempted to name this Point - then it'll be a simpler refactor if we ever move this into a Common directory or something similar.